Document tests/pos/t1168.scala
(1:21..1:24) scala/App#
(3:34..3:43) scala/package.Throwable#
(6:15..6:24) scala/package.Exception#
(6:24..6:24) java/lang/Exception#`<init>`().
(8:14..8:30) _empty_/Test.SpecialException# -> 3:9
(9:7..9:14) scala/Predef.println(+1).
(9:44..9:45) java/lang/String#`+`().
(9:46..9:47) local0 -> 8:10
(10:7..10:13) scala/Predef.assume().
(10:14..10:15) local0 -> 8:10
(10:16..10:28) scala/Any#isInstanceOf().
(10:29..10:45) _empty_/Test.SpecialException# -> 3:9
(12:14..12:23) scala/package.Exception#
(13:7..13:13) scala/Predef.assume().
(13:14..13:15) local1 -> 12:10
(13:16..13:28) scala/Any#isInstanceOf().
(13:29..13:38) scala/package.Exception#

